Buttons.d

```diff
@@ -326,6 +305,29 @@ func int Button_GetCaptionPtr(var int hndl) {
 	return v.textLines_next;
 };
 
+func void _Button_processMouseEvents( var int btn_hndl ) {
```

Comment:
- Change function return type to `int` (for proper stack handling with `foreachHndl`)
- Rename function to `_Buttons_Do_Sub` for consistent style
- Remove white space between parentheses for consistent/readable code

```D
func int _Buttons_Do_Sub(var int btn_hndl) {
```